client/src/App.tsx

```diff
@@ -5,27 +5,24 @@ import { BrowserRouter, Route, Switch } from 'react-router-dom';
 
 import SykmeldingSide from './pages/sykmelding/SykmeldingSide';
 import SykmeldingerPage from './pages/sykmeldinger/SykmeldingerPage';
-import useAppStore from './data/useAppStore';
 
 const App = () => {
     // Endepunkter for å støtte både https://tjenester.nav.no/sykmmleding og https://sykmelding.nais.adeo.no
     return (
-        <useAppStore.Provider>
-            <BrowserRouter>
-                <Switch>
-                    <Route
-                        path={`${process.env.REACT_APP_SYKEFRAVAER_ROOT}/sykmeldinger`}
-                        exact
-                        component={SykmeldingerPage}
-                    />
-                    <Route
-                        path={`${process.env.REACT_APP_SYKEFRAVAER_ROOT}/sykmeldinger/:sykmeldingId`}
-                        component={SykmeldingSide}
-                    />
-                    <Route path="/sykmeldinger/:sykmeldingId" component={SykmeldingSide} />
-                </Switch>
-            </BrowserRouter>
-        </useAppStore.Provider>
+        <BrowserRouter>
+            <Switch>
+                <Route
+                    path={`${process.env.REACT_APP_SYKEFRAVAER_ROOT}/sykmeldinger`}
```

Comment:
Når jeg tenker meg om... trenger vi egentlig å ha en environmentvariabel her? kan ikke se for meg lenken skal endre seg med det første, og da kan vi like gjerne gjøre det i koden

Reply:
Enig. Rettes.